source/collections/Queue.ooc

```diff
@@ -79,8 +79,7 @@ VectorQueue: class <T> extends Queue<T> {
 
 	operator [] (index: Int) -> T {
 		version(safe)
-			if (index < -this count || index >= this count)
-				raise("Indexing in get accessor of VectorQueue outside of range.")
+			raise(index < -this count || index >= this count, "Indexing in get accessor of VectorQueue outside of range.")
```

Comment:
@thomasfanell What do you think, should we put these on one line as we do with `debug` version statements in the other project? These lines might get too long though.